### PR TITLE
Feature/httpsing

### PR DESCRIPTION
**Pull request by ruffrey**
PLS TO REVIEW

* ngrok is included for easier https
* redirects to https on current domain during screenshare

### CODE REVIEW

File: gruntfile.js [hunk 1]
```diff
@@ -201,8 +203,16 @@ module.exports = function (grunt) {
     grunt.loadNpmTasks('grunt-contrib-stylus');
     grunt.loadNpmTasks('grunt-autoprefixer');
     grunt.loadNpmTasks('grunt-gh-pages');
+    
+    grunt.registerTask('grok', 'Fire up ngrok so icanhaz https.', function () {
```
Comment (pho3nixf1re) [inline gruntfile.js]: although it's a nice play on words, i think naming the task `ngrok` would be clearer

**Comment by ruffrey** [comment]
The only question is, how to fix this?

image

**Comment by pho3nixf1re** [comment]
:+1:

File: gruntfile.js (hunk 1 above)
Comment (ruffrey) [inline gruntfile.js]: I was thinking the same thing. There is already a grunt plugin `grunt-ngrok`. However it forces you to use a chosen subdomain, which requires an ngrok account and auth token, and was written around that concept so a PR was not an option. So in the interest of preserving namespace I chose `grok`. I can change to `ngrok` though.

**Comment by chadxz** [comment]
I've been working on this project and i have never seen that error message Jeff. What were you trying to do?

**Comment by chadxz** [comment]
LGTM

**Comment by pho3nixf1re** [comment]
I vote to output the chrome webstore url in the error message.

**Comment by ruffrey** [comment]
@chadxz I was on an ngrok subdomain. Can we `*.ngrok.com` the extension URL or something?

**Comment by ruffrey** [comment]
oh dang @danjenkins 
(missed your comment)

**Comment by chadxz** [comment]
probably so

**Comment by ruffrey** [comment]
Who can edit the chrome web store package?

**Comment by ruffrey** [comment]
image

**Comment by ruffrey** [comment]
I'll wait to merge it until steve gives the verbal OK, at minimum.

**Comment by ssokol** [comment]
Looks good to me. Commit at will.

**Pull request merged**